python-checks/src/main/java/org/sonar/python/checks/PandasDataFrameToNumpyCheck.java

```diff
@@ -23,14 +23,19 @@
 import org.sonar.check.Rule;
 import org.sonar.plugins.python.api.PythonSubscriptionCheck;
 import org.sonar.plugins.python.api.SubscriptionContext;
+import org.sonar.plugins.python.api.quickfix.PythonQuickFix;
 import org.sonar.plugins.python.api.symbols.Symbol;
 import org.sonar.plugins.python.api.tree.QualifiedExpression;
 import org.sonar.plugins.python.api.tree.Tree;
+import org.sonar.python.quickfix.TextEditUtils;
 
 @Rule(key = "S6741")
 public class PandasDataFrameToNumpyCheck extends PythonSubscriptionCheck {
 
+  private static final String DATAFRAME_VALUES_FQN = "pandas.core.frame.DataFrame.values";
   private static final String MESSAGE = "Do not use \"DataFrame.values\".";
+  private static final String QUICK_FIX_MESSAGE = "Replace this with a call to DataFrame.to_numpy()";
```

Comment:
For the message I think we can go with `Replace .values  with DataFrame.to_numpy()`

Reply:
I'd suggest just `Replace with DataFrame.to_numpy()`
I don't remember where exactly are the quick fixes messages guidelines, but I think they mention that we should generally not compare the states before/after and just focus on the state after the fix.

Reply:
Ok good to know!